knotx-core/src/main/java/io/knotx/dataobjects/Fragment.java

```diff
@@ -25,6 +25,9 @@
 import java.util.stream.Collectors;
 import org.apache.commons.lang3.StringUtils;
 
+/**
+ * An entity representing markup fragment that should be processed by Knots.
```

Comment:
An entity representing markup slice produced during Template fragmentation. It represents both markup with static and dynamic content.